HW_1/Task_4.py

```diff
@@ -0,0 +1,12 @@
+# Задача 8: Требуется определить, можно ли от шоколадки размером 
+# n × m долек отломить k долек, если разрешается сделать один разлом по
+# прямой между дольками (то есть разломить шоколадку на два
+# прямоугольника).
+
+a = int(input("Введите количество количество долек по вертикали: "))
+b = int(input("Введите количество количество долек по горизонтали: "))
+c = int(input("Введите количество количество долек которые нужно отломить: "))
+if c < a * b and ((c % a == 0) or (c % b == 0)):
```

Comment:
```suggestion
if c < a * b and (c % a == 0 or c % b == 0):
```
Также можно расширить первое условие по нижней границе допустимых значений переменной `с`